versions.html

```diff
@@ -20,6 +20,9 @@ <h1> PyTorch Documentation </h1>
         <li class="toctree-l1">
           <a class="reference internal" href="main/">main (unstable)</a>
         </li>
+        <li class="toctree-l1">
+          <a class="reference internal" href="0.14/">v0.14 (release candidate)</a>
```

Comment:
We'll need to remember to update that when the release is out for good

Reply:
got it, thanks @NicolasHug , will update this

Reply:
Sorry I just meant the `.0` part i.e. minor version

Reply:
Ah okay, I have fixed it then